backend/baseclass.pm

```diff
@@ -2,6 +2,46 @@
 # Copyright 2012-2021 SUSE LLC
 # SPDX-License-Identifier: GPL-2.0-or-later
 
+# Helper class to identify file descriptors in error case, see do_capture()
+package NamedIOSelect;
```

Comment:
This is better added to a new file.

Reply:
I moved it to a **new** folder `Utils` as I didn't found something else which suites. WDYT?

Reply:
"Utils" is likely just as good as putting it into the top folder but I don't have a better suggestion.

Reply:
The OBS checks fail because you added a new folder which so far is not referenced in the `%files` section of dist/rpm/os-autoinst.spec but I suggest anyway to not introduce a new folder. "Utils" too quickly deteriorates, similar to "misc" ;) Unless other people suggest something different I suggest `backend/`

Reply:
@Martchus why did you suggest to move this to a new file when it's only used within backend/baseclass anyway?

Reply:
Because baseclass is already > 1000 lines and mixing multiple packages in a single file doesn't make it better. I'd just put it under `OpenQA` or under the top-level. I'd avoid `backend` because we don't want to load it as backend via `require "backend/$name.pm";` in `driver.pm`.

Reply:
ok. I also think `OpenQA` is better than `Utils`

Reply:
:+1: